docs/using-the-jdbc-driver/RdsProxy.md

```diff
@@ -0,0 +1,42 @@
+| Scalability   | The AWS JDBC Driver connects through user provided hostnames and does not have a limit on the number of opened connections.                                                                                                                                                                                                                                                                               | The RDS Proxy has a default quota of 20 proxies per region per AWS account. Users may request an increase in the quota.                                                                                                                                                                                                                                            |
+| Flexibility   | If you already have a community JDBC driver fully integrated in your workflow, such as the PostgreSQL JDBC driver or the MySQL Connector/J. The AWS Advanced JDBC Driver allows customers to continue using their existing community drivers in addition to having the AWS JDBC Driver fully exploit failover behavior by maintaining a cache of the Aurora cluster topology and each DB instance's role. | The RDS Proxy provides an additional point of failure to the databases and can act as an additional layer of security between client applications and the underlying database. However the proxies require network connection and may introduce extra complexity during local or offline development.                                                              |
+| Custom DNS    | The AWS JDBC Driver supports custom DNS by specifying the additional connection parameter `clusterInstanceHostPattern`.                                                                                                                                                                                                                                                                                   | The RDS Proxy does not support custom DNS.                                                                                                                                                                                                                                                                                                                         |
```

Comment:
I would avoid mentioning clusterInstanceHostPattern. It adds unnecessary noise to the message.